api/src/main/java/uk/gov/dhsc/htbhf/claimant/SchedulerConfig.java

```diff
@@ -17,6 +18,7 @@
 @Configuration
 @EnableScheduling
 @EnableSchedulerLock(defaultLockAtMostFor = "PT60M")
+@ConditionalOnProperty(name = "enable-scheduler") // annotation has no affect if the property isn't set
```

Comment:
This annotation has not affect if the property isn't set, so no need to set it to true in src/main/resources/application.yml